doc/coding-style.md

```diff
@@ -3,36 +3,62 @@
-   rather than `_member`), the existing style in that file takes precedence.
+   rather than `_member`), the existing style in that file takes precedence, unless one chooses to convert the entire file.
 1. We only use `var` when it's obvious what the variable type is (i.e. `var stream = new FileStream(...)` not `var stream = OpenStandardInput()`).
-1. We use language keywords instead of BCL types (i.e. `int, string, float` instead of `Int32, String, Single`, etc) for both type references as well as method calls (i.e. `int.Parse` instead of `Int32.Parse`). See issue [391](https://github.com/dotnet/corefx/issues/391) for examples.
-1. We use PascalCasing to name all our constant local variables and fields. The only exception is for interop code where the constant value should exactly match the name and value of the code you are calling via interop.
-1. We use ```nameof(...)``` instead of ```"..."``` whenever possible and relevant.
-1. Fields should be specified at the top within type declarations.
+  - Alternatively, in C# 9.0, one can use `Dictionary<string, string> pairs = new();`
+1. We use language keywords instead of BCL types (i.e. `int` instead of `Int32`, etc.) for both type references as well as method calls (i.e. `int.Parse` instead of `Int32.Parse`).
+  > Note: see issue [391](https://github.com/dotnet/corefx/issues/391) for examples.
+1. We use PascalCasing to name all our constant local variables and fields.
+  > Note: the only exception is for interop code where the constant value should exactly match the name and value of the code you are calling via interop.
+1. We use `nameof(...)` instead of `"..."` whenever possible and relevant.
+1. Order of members in a class definition:
+  - Fields (in order: constants, static readonly, static, instance).
+  - Constructors and factory methods (in order: public, protected, internal, private).
+  - Properties (in order: public, protected, internal).
+  - Methods (in order: public, protected, internal, private).
 1. When including non-ASCII characters in the source code use Unicode escape sequences (\uXXXX) instead of literal characters. Literal non-ASCII characters occasionally get garbled by a tool or editor.
-
-> For non code files (e.g. xml, json, etc.) our current best guidance is consistency. When editing files, keep new code and changes consistent with the style in the files.
-For new files, it should conform to the style for that component.
-Last, if there's a completely new component, anything that is reasonably broadly accepted is fine.
-
-## Example Files
+1. Models properties default instances:
+  - Collections should be given a default instance, but if null or empty not be serialized.
+    > Note: while track1, they should also be public settable.
+  - Submodels should not be given a default instance.
+1. Solution folders are used for organizing large numbers of files. We generally aim to keep namespaces to a minimum: the one defined for the project.
```

Comment:
Documented other recent decisions and existing standards not otherwise documented.